floater/generators.py

```diff
@@ -50,6 +71,39 @@ def get_rectmesh(self):
 
         return np.meshgrid(self.x, self.y)
 
+
+    def get_hexcoords(self, mask, mask_grid_x, mask_grid_y):
+        """Get the coordinates of float positions taking into account the land mask.
+
+        RETURNS
+        -------
+        floats_ocean: np.ndarray
+            1D array of float coordinate subarrays: [lat, lon] 
+        """
```

Comment:
how does the user know what shape ``mask_grid_x`` and ``mask_grid_y`` should be?